packages/utils/src/document/reconversion-kickoff-loader.ts

```diff
@@ -0,0 +1,159 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { ReconversionKickoffParams } from "@metriport/core/command/reconversion/reconversion-kickoff-direct";
+import { SQSClient } from "@metriport/core/external/aws/sqs";
+import { getEnvVarOrFail } from "@metriport/core/util";
+import { uuidv7 } from "@metriport/core/util/uuid-v7";
+import { createUuidFromText } from "@metriport/shared/common/uuid";
+import { JSONParser, ParsedElementInfo } from "@streamparser/json";
+import fs from "fs";
+import { chunk, groupBy } from "lodash";
+
+/**
+ * This script is used to kick off a reconversion for a specific list of patients, and for a specific date range.
+ * It will send a message to the reconversion kickoff queue for each patient chunk (recommended to be 10/chunk).
+ *
+ * The reconversion kickoff lambda will then pick up the message and ping the API to reconvert patient documents.
+ *
+ * The input file is generated using the following SQL command:
+ * ```
+ * SELECT patient_id, cx_id
+ * FROM docref_mapping
+ * WHERE created_at BETWEEN <dateFrom> AND <dateTo>;
+ * ```
+ * Once the query is executed, export the results to a JSON file. Then, remove the SQL query from the object, and
+ * only leave the array of objects. Indicate the path to the file in the `fileName` variable.
```

Comment:
Usually easier to work w/ exports as CSV, can use `ingest-patients.ts` as an example/ready to copy code.

NABD if this works though